web/package.json

```diff
@@ -3,8 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "chokidar": "^3.5.1",
```

Comment:
What's this for?

Reply:
if I dont npm install chokidar or w/e, the app doesnt compile. I googled the error and it said to just install the dependency and save it. Should I just remove it?

Reply:
Hmm, looks like a library for file watching? I don't see it as a peerDependency of react-scripts but I suspect that's where it's coming from. Probably a bug, let me try to bump some versions and see if I can fix it...

Reply:
gfdi I hate Facebook, here's the fix:

* Remove chokidar
* Bump react to ^17.0.2
* Bump react-scripts to 4.0.3 (NO carat cuz Facebook likes to publish breaking changes in patch versions :rage:)
* Manually add @ babel/core (no space, GitHub keeps trying to tag the babel user lol) to devDependencies, version ^7.13.0

Reply:
Also change "jsx": "react" to "jsx": "react-jsx" in tsconfig.json for some reason